### ISSUE TITLE
Make use of an empty space more efficiently for a long exception messages

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#34739]
### Description

Sometimes text of an exception is very long.
It becomes difficult to scroll a tiny area to read the message.
Ideally, I suggest to avoid scrolling somehow.
Any additional action that the user has to perform usually reduces the effectiveness of the user's work with the application.

At the same time, a huge area in a tab remains empty.
I understand that it is a placeholder for content of a "Details" panel.
But "Details" area contains an internal overwhelming information about a stack of exception in DBeaver's java-code unwanted for 99% inexperienced users who will never look at DBeaver's sources or pass it to developers for diagnostic reasons.

Another request about ui of this view #34726

### DBeaver Version

Community Edition 24.1.3.202407042322

### Operating System

Windows 10

### Database and driver

PostgreSQL 16
PostgreSQL JDBC Driver 42.7.2

### Steps to reproduce

Use this code to debug the case :

```
DO $$ 
BEGIN 
 RAISE exception 'Some diagnostic info: %', (select string_agg("oid"::text, ', ') from pg_catalog.pg_class);
END $$;
```

https://github.com/dbeaver/dbeaver/assets/33929221/2fcf44d6-a24d-4bc8-842f-dc571f3feaa3

**Comment by E1izabeth** [comment]
Thank you for report.

**Comment by ask9** [comment]
It it fixed as a part of this issue: #34732
